### DISCUSSION TITLE
آی پی غیر کلد فلر

### DISCUSSION THREAD

**Discussion by Arashch405** [MortezaBashsiz/CFScanner#584] [General]
با سلام  واحترام 

آیا میتونم رنج آی پی غیر از رنج آی پی های کلد فار به اسکنر جهت اسکن بدم؟

**Comment by tempookian** [comment]
> با سلام واحترام
> 
> آیا میتونم رنج آی پی غیر از رنج آی پی های کلد فار به اسکنر جهت اسکن بدم؟

می‌تونید اسکنر پایتون رو با فلگ


``
--no-fronting
``


امتحان کنید. احتمالا باید برای G-Core جواب‌گو باشه. اگر بگید برای چه منظوری می‌خواید شاید بهتر بتونم راهنمایی کنم

**Comment by Arashch405** [comment]
سلام مجدد

من میخوام به فرض این رنج آی پی ها
95.179.245.0/24
95.179.243.0/24
95.179.244.0/24
95.179.242.0/24

روکه کلد نیستن با اسکن مرتضی اسکن کنم ...وآی پی تمیزشو بردارم..برای آی پی گلدی ها میشه اما غیر نه

**Reply by tempookian** [reply]
> سلام مجدد
> 
> من میخوام به فرض این رنج آی پی ها 95.179.245.0/24 95.179.243.0/24 95.179.244.0/24 95.179.242.0/24
> 
> روکه کلد نیستن با اسکن مرتضی اسکن کنم ...وآی پی تمیزشو بردارم..برای آی پی گلدی ها میشه اما غیر نه

درود.  منظورتون از «اسکن مرتضی»‌ نسخه بشه؟ عرض من این بود که با نسخه پایتون اسکنر مرتضی باشسیز(cfscanner)  می‌تونید این رنج‌ها رو اسکن کنید. دقت کنید که توی این رنج‌ها در کل تعداد آیپی قابل استفاده خیلی کمن (در حد ۵ ۶ تا بین کلشون). حالا ممکنه همینا هم مسدود شده باشن که در نتیجه در خروجی چیزی نمی‌بینید

**Comment by Arashch405** [comment]
سلام مجدد
منظورم این نسخه ویندوزی بود
https://github.com/MortezaBashsiz/CFScanner/releases/tag/v1.2.8530.30281
خوب برم سراغ نسخه پایتون ببینم میتونم انجامش بدم؟
ویندوز من نسخه7می باشد مشکلی که نداره؟
لینک نسخه پایتون رو ندارین؟

درسته حالا اسکن میکنم  مهم اینه  یاد بگیرم چطوری اسکن کنم
